Cargo.toml

```diff
@@ -50,21 +50,27 @@ opt-level = 1
 [profile.dev.package."*"]
 opt-level = 3
 
-# Enable more optimization in release mode at the cost of compile time.
+# The default profile is optimized for Wasm builds because
+# that's what [Trunk reads](https://github.com/trunk-rs/trunk/issues/605).
+# Optimize for size in wasm-release mode to reduce load times and bandwidth usage on web.
 [profile.release]
 # Compile the entire crate as one unit.
 # Significantly slows compile times, marginal improvements.
 codegen-units = 1
 # Do a second optimization pass over the entire program, including dependencies.
 # Slightly slows compile times, marginal improvements.
 lto = "thin"
-
-# Optimize for size in wasm-release mode to reduce load times and bandwidth usage on web.
-[profile.wasm-release]
-# Use release profile as default values.
-inherits = "release"
 # Optimize with size in mind (also try "s", sometimes it is better).
 # This doesn't increase compilation times compared to -O3, great improvements.
 opt-level = "z"
 # Strip all debugging information from the binary to reduce file size.
 strip = "debuginfo"
+
+# Override some settings for native builds.
```

Comment:
```suggestion
# Enable more optimization in release mode at the cost of compile time.
```

Reply:
Will make more sense to restore this comment when trunk is fixed upstream and this whole workaround is reverted.